Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:21.04 AS build
+
+RUN dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y curl
+
+RUN curl -L -o /usr/local/bin/red https://static.red-lang.org/dl/auto/linux/red-latest && \
+  chmod +x /usr/local/bin/red
```

Comment:
I've been trying to use the official release instead of manually compiling it, which _does_ seem to work, only there is this little message at the end of a Red call (e.g. by running `./bin/run-tests-in-docker.sh`)

```
*** Driver Internal Error: Script Error : status needs a value
*** Where: run-console
*** Near:  [status: either all [Windows? gui?] [
gui-sys-call exe any [all [file form-args file] ""]
]]
```

It does seem to work, so we can ignore this I think.

Reply:
It's related to [exit status PR I made to Red recently](https://github.com/red/red/pull/4869), so this could for example mean that we won't get status `1` on failed tests. I'll tray to debug this.

Reply:
Ah okay. Any idea when there'll be a new release?

Reply:
It's already merged in `master`, and that's what bothers me. Everybody waits for new release, it's over 3 years now :-D Until then we live with latest master and it's fine.

Reply:
Okay, then I know we need to build it ourself.

Reply:
Ok, it's not harmful, although I'm fixing it already: https://github.com/red/red/issues/4889 , probably will be merged soon and available in latest master release.